zds/notification/receivers.py

```diff
@@ -231,12 +231,14 @@ def clean_subscriptions(sender, *, topic, **__):
 
 @receiver(tuto_signals.content_read, sender=ContentReaction)
 @receiver(forum_signals.post_read, sender=Post)
-def mark_comment_read(sender, *, instance, user, **__):
-    comment = instance
+def mark_comments_read(sender, *, instances, user, **__):
+    comments = {}
+    for pk, comment in enumerate(instances):
```

Comment:
pas sûr que pk soit toujours bon, là le enumerate va toujours itérer à partir de 0 jusqu'à la taille de "instances", c'est pas la clef en bdd.

Reply:
Bien vu !